### PR TITLE
Prot rig up port

### LINKED ISSUE #16600: [GENERAL] Protean secondary ears colorless, unremoveable
#### Brief description of the bug
Proteans cannot fully modify their secondary ear slot using their shapeshifting commands

#### What you expected to happen
To be able to freely add, remove and color the additional ears in said slot

#### What actually happened
Any choices made were locked to color FFFFFF, and I couldn't choose Normal after choosing any other (with other uncolorable options like squirrel, red working)

#### Steps to reproduce
- Play Protean
- use the Select Secondary Ears verb to choose any ears
- attempt to use it again to select Normal

#### Code Revision
- 28ce4ab3dffd461829d411ed5649cbbf70ab61b5
 

#### Anything else you may wish to add:
- Character sprite from the round when I found this out, with fox ears secondary ears, kitty ears primary ears
image

### PR DESCRIPTION
**Pull request by Kashargul**
fixes #16600

Up ports proteans from chompstation with rig transformation, hands and some other qols for Id and headset handling. Hand changes, id and headset changes are also added to prommi blobs.

Downstream move list
```
EXISTS (as non _ch versions):
'modular_chomp/icons/mob/species/teshari/suit_ch.dmi': cannot find file
'modular_chomp/icons/mob/head_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/protean/protean.dmi': cannot find file
'icons/obj/rig_modules_ch.dmi': cannot find file
'modular_chomp/icons/mob/uniform_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/protean/protean_powers.dmi': cannot find file

DOES NOT EXIST:
'modular_chomp/sound/voice/borg_deathsound.ogg': cannot find file
'modular_chomp/icons/mob/species/vox/suit_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/xenomorph_hybrid/suit_ch.dmi': cannot find file
'modular_chomp/icons/mob/spacesuit_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/tajaran/helmet_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/skrell/helmet_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/unathi/helmet_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/akula/helmet_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/sergal/helmet_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/vulpkanin/helmet_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/fox/helmet_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/teshari/helmet_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/vox/head_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/xenomorph_hybrid/helmet_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/teshari/feet_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/vox/shoes_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/xenomorph_hybrid/shoes_ch.dmi': cannot find file
'modular_chomp/icons/mob/feet_ch.dmi': cannot find file
'modular_chomp/icons/mob/hands_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/teshari/hands_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/vox/gloves_ch.dmi': cannot find file
'modular_chomp/icons/mob/species/xenomorph_hybrid/gloves_ch.dmi': cannot find file
'modular_chomp/icons/mob/lefthand_holder.dmi': cannot find file
'modular_chomp/icons/mob/righthand_holder.dmi': cannot find file
'modular_chomp/icons/mob/suit_ch.dmi': cannot find file
'modular_chomp/icons/mob/ears.dmi': cannot find file
'modular_chomp/icons/obj/slimeborg/slimecore.dmi': cannot find file
'modular_chomp/icons/mob/species/protean/protean64x32.dmi': cannot find file
'modular_chomp/icons/mob/species/protean/protean64x64.dmi': cannot find file
'modular_chomp/icons/mob/species/protean/protean128x64.dmi': cannot find file
```

Downstream:
Date for us needs to be changed back in code/modules/mob/living/carbon/human/species/station/protean_vr/protean_species.dm

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/station/protean_vr/protean_species.dm [hunk 1]
```diff
@@ -104,13 +110,29 @@
 		/mob/living/carbon/human/proc/shapeshifter_select_wings,
 		/mob/living/carbon/human/proc/shapeshifter_select_tail,
 		/mob/living/carbon/human/proc/shapeshifter_select_ears,
-		/mob/living/carbon/human/proc/shapeshifter_select_secondary_ears,
```
Comment (Kashargul) [inline code/modules/mob/living/carbon/human/species/station/protean_vr/protean_species.dm:107]: todo me, fix that

File: code/modules/mob/living/carbon/human/species/station/protean_vr/protean_species.dm [hunk 2]
```diff
@@ -172,23 +238,28 @@
...
+			H.nano_rig_transform(1)
+	else
+		H.nano_rig_transform(1)
+	pseudodead = 1
 
 /datum/species/protean/handle_environment_special(var/mob/living/carbon/human/H)
 	if((H.getActualBruteLoss() + H.getActualFireLoss()) > H.maxHealth*0.5 && isturf(H.loc)) //So, only if we're not a blob (we're in nullspace) or in someone (or a locker, really, but whatever)
 		return ..() //Any instakill shot runtimes since there are no organs after this. No point to not skip these checks, going to nullspace anyway.
 
+/*CHOMP Station removal start
```
Comment (Kashargul) [inline code/modules/mob/living/carbon/human/species/station/protean_vr/protean_species.dm:262]: likely fully delete this section

File: code/modules/mob/living/carbon/human/species/station/protean_vr/protean_species.dm [hunk 3]
```diff
@@ -350,13 +434,11 @@
 	owner = 1
 	if(new_name)
 		name += " ([new_name])"
-		validstring += "[time2text(world.timeofday, "Month") +" "+ num2text(text2num(time2text(world.timeofday, "YYYY"))+300)]"
+		validstring += "[time2text(world.timeofday, "Month") +" "+ num2text(text2num(time2text(world.timeofday, "YYYY"))+544)]" //YW EDIT
```
Comment (Kashargul) [inline code/modules/mob/living/carbon/human/species/station/protean_vr/protean_species.dm:437]: fix that for +300 on virgo, refix on chomp later

**Comment by TankTheBirb** [comment]
```
*/icons/mob/lefthand_holder.dmi <-- exists in icons/mob/items/lefthand_holder.dmi but lacks puddle0, puddle1, puddle2
*/icons/mob/righthand_holder.dmi <-- exists in icons/mob/items/lefthand_holder.dmi but lacks puddle0, puddle1, puddle2
*/icons/mob/uniform_ch.dmi <-- exists, icons/mob/uniform.dmi. Missing icon states vrsuit, puddle0, puddle1, puddle2
*/icons/mob/suit_ch.dmi <-- does not exist unless this was meant to be the teshari one? or is more dispersed in many files. anyways there's just the puddle0, puddle1, puddle2 sprite states there
*/icons/mob/ears.dmi <-- exists as mob/human_races/sprite_accessories/ears.dmi but lacks puddle0, puddle1, puddle2
icons/obj/rig_modules_ch.dmi <-- does exist as rig_modules.dmi and rig_modules_vr.dmi but lacks icon state nanomachine_rig
```
I'll do more later but these seemed like the first and easiest to poke at for the sprite port side of things

**Comment by SatinIsle** [comment]
I am seeing the following runtime when going in and out of blobform:
```

[2024-11-15T13:19:31] Runtime in code/modules/vore/resizing/resize_vr.dm,71: Cannot read null.limit_mob_size
  proc name: has large resize bounds (/atom/movable/proc/has_large_resize_bounds)
  usr: Mr Protean (/mob/living/simple_mob/protean_blob) @ (the floor) (19,20,1) (/turf/simulated/floor/tiled) (ckey=satinisle)
  usr.loc: (the floor) (19,20,1) (/turf/simulated/floor/tiled)
  src: Mr Protean (/mob/living/carbon/human)
  src.loc: null
  call stack:
  Mr Protean (/mob/living/carbon/human): has large resize bounds()
  Mr Protean (/mob/living/carbon/human): resize(0.25, 0, 1, 1, 1)
  Mr Protean (/mob/living/carbon/human): resize(0.25, 0, 1, 1, 1)
  Mr Protean (/mob/living/carbon/human): nano outofblob(Mr Protean (/mob/living/simple_mob/protean_blob), null)
  Mr Protean (/mob/living/carbon/human): Toggle Blobform(null)
```

**Pull request merged**